src/components/maps/_control-layers.scss

```diff
@@ -0,0 +1,75 @@
+@import "../../helpers/all";
+@import "../../settings/all";
+
+.smbc-control-layers.leaflet-control {
+  border-radius: 5px;
+  background: govuk-colour("white");
+  box-shadow: 0 1px 5px rgba(0, 0, 0, .4); // sass-lint:disable-line no-color-literals
+
+  .smbc-control-layers__group {
+    .smbc-control-layers__group-header {
+      font-weight: bold;
```

Comment:
We should use the GOV UK DS variable for font weight bold in case we change the default bold in the future, all instances are then in line: https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/settings/_typography-font.scss#L89